### PR TITLE
update CMakeLists.txt to 3.5 and fix modplug search

### PR DESCRIPTION
**Pull request by AlwinEsch**

### CODE REVIEW

**Comment by Rechi** [comment]
Using `and` in a commit message title is always an indicator that you actually should split the changes into two (or more) commits.

File: FindModplug.cmake [hunk 1]
```diff
@@ -9,6 +9,7 @@ find_package(PkgConfig)
 if(PKG_CONFIG_FOUND)
   pkg_check_modules (MODPLUG libmodplug)
   list(APPEND MODPLUG_INCLUDE_DIRS ${MODPLUG_INCLUDEDIR})
+  link_directories(${MODPLUG_LIBRARY_DIRS})
```
Comment (Rechi) [inline FindModplug.cmake]: If you want to fix this CMake find module, then do it the right way by never trusting pkg-config only.

**Comment by AlwinEsch** [comment]
Is updated, about last commit, make a version increase? There it seems to have never worked before by jenkins.

**Comment by Rechi** [comment]
I'll take care of the version bump and releasing a new version once this PR is merged.

**Comment by AlwinEsch** [comment]
@Rechi is merged.

**Pull request merged**